projects/kit/components/input-date/multi-date/input-multi-date.template.html

```diff
@@ -0,0 +1,85 @@
+<tui-hosted-dropdown
+    class="t-hosted"
+    [canOpen]="canOpen"
+    [content]="dropdown"
+    [open]="open && canOpen"
+    (openChange)="onOpenChange($event)"
+>
+    <tui-input-tag
+        *tuiLet="filler$ | async as filler"
+        automation-id="tui-input-date-tag__textfield"
+        class="t-input"
+        [disabled]="disabled"
+        [disabledItemHandler]="disabledItemHandler | tuiMapper: disabledItemHandlerWrapper"
+        [editable]="editable"
+        [expandable]="expandable"
+        [inputHidden]="!editable"
+        [nativeId]="nativeId"
+        [ngModel]="value"
+        [placeholder]="placeholder"
+        [pseudoFocus]="pseudoFocus"
+        [pseudoHover]="pseudoHover"
+        [pseudoInvalid]="computedInvalid"
+        [readOnly]="readOnly"
+        [removable]="!nativeDropdownMode"
+        [rows]="rows"
+        [tagValidator]="tagValidator | tuiMapper: disabledItemHandlerWrapper"
+        [tuiTextfieldFiller]="getComputedFiller(filler || '')"
+        [tuiTextfieldIcon]="calendarIcon && iconContent"
+        (ngModelChange)="onValueChange($event)"
+    >
+        <ng-content></ng-content>
+        <ng-content
+            ngProjectAs="input"
+            select="input"
+        ></ng-content>
+    </tui-input-tag>
+
+    <ng-template #iconContent>
+        <tui-svg
+            *polymorpheusOutlet="calendarIcon as src; context: {$implicit: size}"
+            appearance="icon"
+            automation-id="tui-input-date-range__icon"
+            tuiWrapper
+            [class.t-icon]="!computedDisabled"
+            [src]="src"
+            (click)="onIconClick()"
+        ></tui-svg>
+        <input
+            *ngIf="isMobile && nativePicker"
+            tuiDate
+            class="t-native-input"
+        />
+    </ng-template>
+
+    <ng-template
+        #dropdown="polymorpheus"
+        [polymorpheus]="type"
+    >
+        <tui-calendar
+            automation-id="tui-input-date__calendar"
+            tuiPreventDefault="mousedown"
+            [disabledItemHandler]="disabledItemHandler"
+            [markerHandler]="markerHandler"
+            [max]="computedMax"
+            [min]="computedMin"
+            [month]="computedActiveYearMonth"
+            [value]="value"
+            (dayClick)="onDayClick($event)"
+            (monthChange)="onMonthChange($event)"
+        ></tui-calendar>
+        <div
+            *ngIf="items.length === 1"
+            tuiPreventDefault="mousedown"
+            class="t-button"
+        >
+            <button
+                tuiLink
+                type="button"
+                (click)="onDayClick(items[0].day)"
```

Comment:
This shouldn't be a named date, it should just be "Done" or "Select" or something like that to close the dropdown.

Reply:
fixed